css/overrides.scss

```diff
@@ -0,0 +1,80 @@
+/*
+sass/config/variables stuff
+ */
+$font-prefix: '../node_modules/@ibm/plex/';
+@import 'node_modules/@ibm/plex/scss/ibm-plex.scss';
+// these sass variables aren't all working (at least color isn't)
+$color-primary: #0050d8;
+$font-sans: 'IBM Plex Sans', sans-serif;
+
+// not sure if these need to be here?
+// @import "../node_modules/us-forms-system/src/scss/base/_b-variables";
+// @import '../node_modules/uswds/src/stylesheets/core/variables';
+
+@import '../node_modules/us-forms-system/lib/css/styles.css';
+
+/*
+general/global overrides,
+these might be bad depending where the form ends up
+*/
+
+/* input style dosen't affect description widget atm */
+input,
+textarea {
+  border-radius: 4px;
+  border-color: #adadad;
+}
+
+input[type='radio'] {
+  border-color: #dbdee1;
+  opacity: 1;
+}
+
+[type='checkbox'] + label,
+[type='radio'] + label {
+  padding: 10px 13px 11px 12px;
+}
+
+[type='radio'] + label::before {
+  box-shadow: 0 0 0 2px #ffffff, 0 0 0 3px #dbdee1;
+}
+
+label {
+  margin-top: 0px;
+}
+
+/*
+form overrides
+ */
+
+[class^='schemaform'] {
+  font-family: $font-sans;
+}
+
+.schemaform-widget-wrapper {
+  margin-top: 46px;
+}
+
+// affect nested children while hopefully avoiding future collision probs
+[class^='schemaform'] > body,
```

Comment:
I tried commenting these out and it seems like it's still working, do we need both this and the one on line 50?

Reply:
nevermind, shows up on the intro page